main/init/package.mill

```diff
@@ -63,8 +64,16 @@ object `package` extends RootModule with build.MillPublishScalaModule {
   }
 
   object buildgen extends build.MillPublishScalaModule {
-    def moduleDeps = Seq(build.runner)
+    def moduleDeps = Seq(build.runner /*, tree(build.Deps.scalaVersion)*/ )
     def testModuleDeps = super.testModuleDeps ++ Seq(build.scalalib)
+
+    // I tried moving `Tree` into this module, but it doesn't compile with Scala 2.12.20.
```

Comment:
Could you elaborate on what this is about? I thought this module should be compiled with Scala 3.6.3